code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -288,73 +287,92 @@
 			replace_beaker(usr)
 			. = TRUE
 		if("dispense_recipe")
-			if(!is_operational || QDELETED(cell))
-				return
-
-			var/list/chemicals_to_dispense = saved_recipes[params["recipe"]]
-			if(!LAZYLEN(chemicals_to_dispense))
```

Comment:
Why did you remove this?

Reply:
remove what

Reply:
You mean if(!LAZYLEN(chemicals_to_dispense))? Mainly because it doesn't seem to actually do anything?